ROADMAP.md

```diff
@@ -33,33 +32,13 @@
 #### build
 - [ ] cmake: add openmp parallelization
 
-#### `templating`
-- [x] remove duplicated code to generate external functions
-- [ ] add documentation
-  - [x] how to set up trenderer
-  - [ ] ...
-- [ ] explore possibility to interact with generated C code
-- [ ] set up CI tests - matlab needed..
-- [ ] add GNSF
-- [ ] make compatible with octave (if possible) - jsonencode alternative?!
-
 #### `ocp_nlp`
 - [x] Gauss-Newton SQP
 - [x] exact Hessian SQP
 - [ ] partial tightening <!-- - [ ] HPNMPC (what?!) -->
 - [ ] blockSQP
 - [ ] RTI implementation similar to ACADO
 
-#### `ocp_qp`
-- [x] qpOASES v3.1
-- [x] OOQP
-- [x] qpDUNES
-- [x] HPMPC
-- [x] OSQP
-
 #### `sim`
-- [x] explicit Runge-Kutta
-- [x] lifted IRK
```

Comment:
lifted IRK are kind of lagging behind in terms of interfacing aren't they? :)

Reply:
right!
This paragraph was just more about features in the core.